packages/extension-ui/src/Popup/Accounts/EditName.tsx

```diff
@@ -0,0 +1,95 @@
+// Copyright 2019-2023 @polkadot/extension-ui authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { ThemeProps } from '../../types';
+
+import React, { useCallback, useContext, useState } from 'react';
+import { RouteComponentProps, withRouter } from 'react-router';
+import styled from 'styled-components';
+
+import { AccountContext, ActionContext, Address, Button, ButtonArea, VerticalSpace } from '../../components';
+import useToast from '../../hooks/useToast';
+import useTranslation from '../../hooks/useTranslation';
+import { editAccount } from '../../messaging';
+import { Header, Name } from '../../partials';
+
+interface Props extends RouteComponentProps<{ address: string }>, ThemeProps {
+  className?: string;
+}
+
+function EditName({
+  className,
+  match: {
+    params: { address }
+  }
+}: Props): React.ReactElement<Props> {
+  const { t } = useTranslation();
+  const { show } = useToast();
+  const { accounts } = useContext(AccountContext);
+  const onAction = useContext(ActionContext);
+
+  const _goTo = useCallback((path: string) => () => onAction(path), [onAction]);
```

Comment:
Probably again a matter of this repo's _convention_, but I don't believe wrapping every function in `useCallback` by default contributes to performance gains. It does, however, contribute to making code less readable.

Reply:
This is the pattern used by polkadot in all their files and enforced by the eslint. I don't find this pattern readable as well but based on the es-lint rule that enforces this, it seems to have performance benefits. https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md

Reply:
I see, that explains it.

Just as a note: that eslint rule is quite far-fetched - I've measured components performance numerous times and the regular function was almost never a problem until it was pretty obvious it was a problem, so from my experience it's completely safe not to use `useCallback` by default, especially for the readability advantage.

Reply:
@jalooc I'm not saying I agree with the rule 😄  Even if there was a slight performance benefit for doing it this way, I would never choose to write it in such a manner for the same reasons you brought up :)